src/main/kotlin/org/stt/text/JiraExpansionProvider.kt

```diff
@@ -17,23 +17,21 @@ constructor(private val jiraConnector: JiraConnector,
             queryText = text.substring(spaceIndex, text.length).trim { it <= ' ' }
         }
 
-        return emptyList()
-
-//        return try {
```

Comment:
here, too - commented out code

Reply:
It's the old version of the file. But comment-out code will be in use again.